src/main/resources/db/changeLog.xml

```diff
@@ -42,4 +42,25 @@
             <column name="modifiedat" type="datetime(6)"/>
         </createTable>
     </changeSet>
+
+    <changeSet id="projectthemes" author="katja danilova">
+        <createTable tableName="projecttheme">
+            <column name="id" type="binary(16)">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="logourl" type="varchar(512)"> <!--todo len -->
+                <constraints nullable="false"/>
+            </column>
+            <column name="themeColor" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="project_id" type="binary(16)">
+                <constraints nullable="false" foreignKeyName="FK_PROJECTHTEME_PROJECT_ID" referencedColumnNames="id" referencedTableName="project"/>
+            </column>
+            <column name="creatorid" type="binary(16)"/>
+            <column name="lastmodifierid" type="binary(16)"/>
+            <column name="createdat" type="datetime(6)"/>
+            <column name="modifiedat" type="datetime(6)"/>
```

Comment:
Is there a typo in the column names? Does it matter?

Reply:
no, it is correct names :) just lowercase